docs/Configuration.md

```diff
@@ -147,7 +147,7 @@ exporters:
 
 #### Pre-built custom images
 
-This method can be used to deploy Pelorus with the user built images or pre-built images mirrored in other than [quay.io](quay.io) registry.
+This method can be used to deploy Pelorus with the user built images or pre-built images mirrored in other than [quay.io](https://quay.io/) registry.
```

Comment:
This kind of errors, the `mkdocs` warns us. I created a issue to try to enforce this in our CI